bin/check_tron_jobs.py

```diff
@@ -367,6 +375,9 @@ def main():
     client = Client(args.server)
 
     error_code = 0
+
+    global _run_interval
```

Comment:
I think this is overkill. 

How often are we actually going to change this number? I bet never and think it is ok to be a constant hard-coded value.

Reply:
Maybe one day we want to set it as one minute to get alerting earlier 🤔